azure-pipelines.yml

```diff
@@ -240,7 +238,7 @@ jobs:
       set -ev
       . scripts/ci/install.sh
       # verify azdev linter works
-      azdev linter acr
+      azdev linter CLI
```

Comment:
This could take a longer time than current. Is that acceptable?

Reply:
Based on the offline discussion, we agree it's acceptable.